src/common/lambdaConfig.go

```diff
@@ -0,0 +1,92 @@
+package common
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v3"
+)
+
+var LambdaConf *LambdaConfig
```

Comment:
I global makes sense for the worker config (because there is one per process), but we will want different configs for each lambda function.

Currently, each function has sandbox metadata (that gets populated based on requirements.txt) here: https://github.com/open-lambda/open-lambda/blob/main/src/worker/lambda/lambdaFunction.go#L30.  Perhaps we should have a FunctionMeta that includes both a SandboxMeta and a LambdaConfig?  Or something like that?

This function (https://github.com/open-lambda/open-lambda/blob/main/src/worker/lambda/lambdaFunction.go#L71) could possibly load both: your LambdaConfig (from an ol.yaml) and SandboxMeta (from a SandboxMeta).